Find a functionally equivalent code:android.widget.CalendarView.getWeekDayTextAppearance:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the text appearance for the week day abbreviation of the calendar header.
 *
 * @return The text appearance resource id.
 *
 * @attr ref android.R.styleable#CalendarView_weekDayTextAppearance
 */

@@ -1,3 +1,3 @@
 {
-    return mWeekDayTextAppearanceResId;
+    return mDelegate.getWeekDayTextAppearance();
 }

